dts/arm/st/f4/stm32f429.dtsi

```diff
@@ -1,90 +1,7 @@
 /*
- * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2020 Linaro Limited
```

Comment:
Don't remove an existing copyright. Change to:
```
Copyright (c) 2017, 2020 Linaro Limited
```

Reply:
fixed